gen-ai/inference/vllm-rayserve-gpu/ray-service-vllm.yaml

```diff
@@ -78,12 +78,19 @@ spec:
               name: ray-logs
             resources:
               limits:
-                cpu: 16
-                memory: "32G"
+                cpu: 1
+                memory: "6G"
               requests:
-                cpu: 16
-                memory: "32G"
+                cpu: 1
+                memory: "6G"
             env:
+            - name: RAY_GRAFANA_HOST
+              value: FILLIN
+            - name: RAY_PROMETHEUS_HOST
+              value: >-
+                FILLIN
+            - name: RAY_GRAFANA_IFRAME_HOST
+              value: FILLIN
```

Comment:
This blueprint also deploys Kube Prometheus stack. Can we update these to localhost services with the port?

Reply:
I don't know what those are for your deployment. I'll let @ratnopamc add them